password/src/Data/Password/Validate.hs

```diff
@@ -79,18 +81,31 @@ defaultPasswordPolicy = PasswordPolicy
     uppercaseChars = Just 1,
     lowercaseChars = Just 1,
     specialChars = Nothing,
-    digitChars = Just 1,
-    charSet = defaultCharSet
+    digitChars = Just 1
   }
 
+-- | Predicate which defines the characters that can be used for a password.
+newtype CharSetPredicate = CharSetPredicate
+  { getCharSetPredicate :: Char -> Bool
+  }
```

Comment:
Introduce `CharSetPredicate` which defines the characters that can be used for a password. Also, `charSet` was removed from `PasswordPolicy`. Due to this change, function `isValidPassword` and `validatePassword` now takes an extra argument `CharSetPredicate`.